### PR TITLE
Added cleanDescendants to match cleanNode and applyBindingsToDescendants

### LINKED ISSUE #1621: Add cleanDescendants
There is a `ko.cleanNode(node)` method for cleaning up after `ko.applyBindings(vm, node)`, but there is no equivalent to `ko.applyBindingsToDescendants(vm, node)`. I propose a `ko.cleanDescendants(node)` method that will only clean up the descendants of the node, not the node itself.

### PR DESCRIPTION
**Pull request by mariusGundersen**
This fixes #1621

### CODE REVIEW

File: src/utils.domNodeDisposal.js [hunk 1]
```diff
@@ -85,7 +90,7 @@ ko.utils.domNodeDisposal = new (function () {
                 node.parentNode.removeChild(node);
         },
 
-        "cleanExternalData" : function (node) {
+        cleanExternalData : function (node) {
```
Comment (SteveSanderson) [inline src/utils.domNodeDisposal.js]: Does this un-export `cleanExternalData`, and is that deliberate? I thought this was an intentional extensibility point.

Comment (mariusGundersen) [inline src/utils.domNodeDisposal.js]: I just found it weird that this name was quoted when none of the others were. Should it be in the ko.exportSymbol list at the bottom of the file?

**Comment by mariusGundersen** [comment]
Reverted the change that probably had unintended consequenses

**Comment by mbest** [comment]
I've merged a version of this change now.

**Pull request closed without merging**